### PR TITLE
CS-32

### PR DESCRIPTION
**Pull request by kceddinger**
Implements caseworker flow with available components

### CODE REVIEW

**Review by jonathanbutler7 — CHANGES_REQUESTED** [review]
Kimberly this looks awesome! I just left you a couple thoughts.

My bad for giving bad instructions in the Jira ticket on how to implement <Link>. That was how I had always done it before, but I hadn't considered this specific use case that all the buttons in the mockups are actually links to other pages. Hopefully I'm making sense, let me know if I'm being confusing lol. I left you an example and a link to the materialUI docs of how to use their buttons with react-router's api. It's pretty cool that they got that integration so seamless IMO. 

Please let me know if you have any questions or further thoughts!

File: src/components/Welcome/DashboardBody.js [hunk 1]
```diff
@@ -18,10 +24,18 @@ export default function DashboardBody() {
         customize your meeting types.
       </Typography>
       <div className={classes.buttonContainer}>
-        <Button variant='contained' color='primary'>
-          let&apos;s get started
-        </Button>
+        <Link to="/scheduledays">
```
[the remark this answers is not in the document]
Comment (kceddinger) [reply src/components/Welcome/DashboardBody.js:27]: Awesome! I was curious about the link's effect on the button. I'll try this out.

File: src/components/Welcome/DashboardBody.js [hunk 2]
```diff
@@ -18,10 +24,18 @@ export default function DashboardBody() {
...
+          <Button variant='contained' color='primary'>
+            let&apos;s get started
+          </Button>
+        </Link>
       </div>
+      <Switch>
```
[the remark this answers is not in the document]
Comment (kceddinger) [reply src/components/Welcome/DashboardBody.js:33]: Early into my implementation I thought each use of the router needed a switch statement. I thought I had removed all of these - nice catch, I'll get rid of it!

File: src/App.js [hunk 3]
```diff
@@ -19,8 +22,10 @@ import ContactReview from './components/Scheduling/ContactReview';
 
 function App() {
   return (
+    <Router>
```
[the remark this answers is not in the document]
Comment (kceddinger) [reply src/App.js:25]: Oh I didn't realize that, thank you! Removing.

File: src/components/Success/SuccessScreenTwo.js [hunk 4]
```diff
@@ -17,10 +24,18 @@ export default function SuccessScreenTwo() {
...
       </div>
       <div className={classes.SuccessScreenTwoButton}>
+      <Link to="/meetingtypes">
         <Button variant='contained' color='primary'>
           Complete Setup
         </Button>
+      </Link>
       </div>
     </div>
+    <Switch>
```
[the remark this answers is not in the document]
Comment (kceddinger) [reply src/components/Success/SuccessScreenTwo.js:34]: I was mistaken and didn't completely eliminate the switch/route statements from other components. My bad!

**Review by jonathanbutler7 — APPROVED** [review]
Looks great

**Pull request merged**